controls/update_def.cf.in

```diff
@@ -128,6 +128,12 @@ bundle common update_def
       comment => "Group that CFEngine Enterprise webserver runs as",
       handle => "common_def_vars_cf_cfapache_group";
 
+      "mpf_update_policy_master_location" -> { "ENT-3692" }
+        comment => "Directory where clients should get policy from.",
+        string => ifelse( isvariable( $(def.mpf_update_policy_master_location) ),
+                                      $(def.mpf_update_policy_master_location),
+                          "masterfiles");
```

Comment:
Shouldn't this use ``"sys.masterdir"`` instead of just ``"masterfiles"`` though?

Reply:
This "masterfiles" shortcut has been in the policy framework since 3.7.0. It's more portable than `$(sys.masterdir)`, since `$(sys.masterdir)` is set at compile time.
- Other people package with FHS so masterdir is `/etc/cfengine/masterfiles` in some builds
- On windows `$(sys.masterdir)` is `C:/Program Files/cfengine/masterfiles` or something similar.

Having clients request `masterfiles` instead of an explicit path allows the server to determine where masterfiles should come from. Note that we already use `masterfiles` shortcut for bootstrap operations.

Reply:
Thanks for the explanation.